### ISSUE TITLE
WCAG 2.4.4 🟠 (Partially Supports): Link Purpose (In Context)

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#5149]
**Level:** A
**Principle:** Operable
**Result:** 🟠 Partially Supports

### Resources
[WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context.html)
> The [purpose of each link](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context#dfn-link-purpose) can be determined from the link text alone or from the link text together with its [programmatically determined link context](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context#dfn-programmatically-determined-link-context), except where the purpose of the link would be [ambiguous to users in general](https://www.w3.org/WAI/WCAG22/Understanding/link-purpose-in-context#dfn-ambiguous-to-users-in-general).

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [x] Determine compliance status, add this to issue name.
- [x] Add any new issues to the Accessibility project for triage.

### Previous audit

❌ #4131 
Many links were contextual, but there is work to be done to become fully complaint here, in part because online articles have followed their print tradition where referencing includes full address details for web-links. The audit also identified that ISSN numbers were being interpreted as telephone numbers by iOS Safari, and automatically being turned into links to phone that number.

 ### Progress since last audit

- #3922
- #3939
- #4083
- #4149
- #4220
- #4221
- #4228
- #4287
- #4650
- #4864
- #4865

### Work Remaining

- #3008
- #4084
- #4138
- #4335
- #4837

### Automated Testing
When testing for link-name, which finds out whether links ahave discernible text, both OLH and Clean passed, but material a single failure on every page, which means the majority of it passed.

 ### Audit Notes

The automated testing only found failures on Material, but we also have identified issues that relate to our processing of XML which will affect all themes.  Some failures are due to user content and as described in #4837 we need to provide more guidance on how to prevent these.

 ### Work arising
- #5245
